packages/utils/src/lib/reports/generate-md-report-category-section.unit.test.ts

```diff
@@ -10,14 +11,6 @@ import { ScoredGroup, ScoredReport } from './types';
 // === Categories Overview Section
 
 describe('categoriesOverviewSection', () => {
-  it('should skip categories table if categories are empty', () => {
```

Comment:
🧪 Unit test coverage: By removing this test, the category section no longer has unit tests (in the `generate-md-report.ts`, there is no test focused on category section).

Reply:
The `generateMdReport` function already check categories length and wouldn't call `categoriesOverviewSection` if they were empty, so I removed the duplicate condition.

Regarding the test, the no categories appears only in an integration test, which I don't think is the best place for it, as it creates another largely duplicated snapshot. Removed the integration test and created a unit test instead.